src/scss/_theme.scss

```diff
@@ -6,7 +6,7 @@
 		text-align: center;
 		-webkit-tap-highlight-color: transparent;
 
-		[class*='owl-'] {
+		button.owl-prev, button.owl-next {
```

Comment:
Is this still working with the button elements? I don't know why this level operator button.owl* was introduced, but could imagine that otherwise these styles might not apply in all cases?

Reply:
Correct: `[class*='owl-']` is overwritten by https://github.com/OwlCarousel2/OwlCarousel2/blob/b6a364c048d1290fe2363760b2cf0fa31e55f47a/src/scss/_core.scss#L76-L77 due to CSS selector precedence